website/docs/understanding-ferretdb.md

```diff
@@ -4,20 +4,54 @@ sidebar_position: 2
 
 # Understanding FerretDB
 
-FerretDB is a document database that uses similar commands, drivers, and tools to those of MongoDB for storing data.
-Unlike relational databases, which use tables, rows, and columns to define the schema of the database, document databases use key-value pairs to build the structure of a document.
+FerretDB is an open source MongoDB alternative that converts MongoDB wire protocols to SQL, with a backend on PostgreSQL.
+As a document database, FerretDB uses similar commands, drivers, and tools to those of MongoDB.
```

Comment:
I think "similar" is not the right word there: we don't use "similar" drivers – we use MongoDB drivers as-is